src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -140,9 +140,9 @@ class OwlViTObjectDetectionOutput(ModelOutput):
         class_embeds (`torch.FloatTensor` of shape `(batch_size, num_patches, hidden_size)`):
             Class embeddings of all image patches. OWL-ViT represents images as a set of image patches where the total
             number of patches is (image_size / patch_size)**2.
-        text_model_last_hidden_states (`torch.FloatTensor` of shape `(batch_size, sequence_length, hidden_size)`)):
+        text_model_last_hidden_state (`torch.FloatTensor` of shape `(batch_size, sequence_length, hidden_size)`)):
```

Comment:
This is not backwards compatible, right? 

But as OWL-ViT is a pretty new model, it seems fine.

Reply:
That's true, it used to be `text_model_last_hidden_states` but they are the last hidden states of the base CLIP model. Since we don't support training yet, I don't think it'd be much of a problem